pkg/login/social/connectors/org_role_mapper.go

```diff
@@ -54,6 +61,11 @@ func (m *OrgRoleMapper) MapOrgRoles(
 	externalOrgs []string,
 	directlyMappedRole org.RoleType,
 ) map[int64]org.RoleType {
+	if mappingCfg == nil {
+		// Org mapping is not configured
+		return m.getDefaultOrgMapping(false, directlyMappedRole)
+	}
```

Comment:
I'm curios why the first parameter from `m.getDefaultOrgMapping` func is:
- `false` when mappingCfg == nil
- `mappingCfg.strictRoleMapping` when len(mappingCfg.orgMapping) == 0 (see lines 69-72)

Reply:
That was a case that shouldn't really happen, but wanted to ensure this. However, I have just changed the code to accept a struct instead of a pointer to make the intention clear.